spec/unleash/constraint_spec.rb

```diff
@@ -384,24 +388,21 @@
       expect(constraint.matches_context?(context)).to be false
     end
 
-    it 'rejects constraint construction for invalid value types for operator' do
-      array_constraints = ['STR_CONTAINS', 'STR_ENDS_WITH', 'STR_STARTS_WITH', 'IN', 'NOT_IN']
-
-      array_constraints.each do |operator_name|
-        Unleash::Constraint.new('env', operator_name, [])
-        expect do
-          Unleash::Constraint.new('env', operator_name, '')
-        end.to raise_error
-      end
-
-      string_constraints = ['NUM_EQ', 'NUM_GT', 'NUM_GTE', 'NUM_LT', 'NUM_LTE',
-                            'DATE_AFTER', 'DATE_BEFORE', 'SEMVER_EQ', 'SEMVER_GT', 'SEMVER_LT']
-      string_constraints.each do |operator_name|
-        Unleash::Constraint.new('env', operator_name, '')
-        expect do
-          Unleash::Constraint.new('env', operator_name, [])
-        end.to raise_error
-      end
+    it 'handles malconfiguration of constraint operators' do
```

Comment:
```suggestion
    it 'handles gracefully invalid constraint operators' do
```

Reply:
What about `it 'gracefully handles invalid constraint operators'` instead? I read this as handling gracefully invalid operators and I'm not immediately sure how something can be gracefully invalid

Reply:
Good point! Your suggested wording is even better/clearer! Love it!

Reply:
done